src/ui/components/TimeSystemAxis.vue

```diff
@@ -139,7 +143,7 @@ export default {
       }
     },
     setDimensions() {
-      const axisHolder = this.$refs.axisHolder;
+      const axisHolder = this.axisHolder;
```

Comment:
super nitpicky, but can we just use `this.axisHolder` here instead of creating the new const?